core/src/consensus/tendermint/stake/action_data.rs

```diff
@@ -41,12 +41,12 @@ pub struct StakeAccount<'a> {
 }
 
 impl<'a> StakeAccount<'a> {
-    pub fn load_from_state(state: &TopLevelState, address: &'a Address) -> StakeResult<StakeAccount<'a>> {
+    pub fn load_from_state(state: &TopLevelState, address: &'a Address) -> StateResult<StakeAccount<'a>> {
```

Comment:
Why did you change all `StakeResult` to `StateResult`?
How about using `StakeResult` in the stake action handler and converting them to `StateResult` in `Stake::init()` and `Stake::execute()`?

Reply:
I think it is not feasible to simply convert a type `StakeResult` into the external type `StateResult`